extension/src/experiments/model/index.ts

```diff
@@ -81,6 +83,10 @@ export class ExperimentsModel extends ModelWithPersistence {
       PersistenceKey.EXPERIMENTS_STATUS,
       {}
     )
+    this.starredExperiments = this.revive<StarredExperiments>(
```

Comment:
[Q] What will happen if we move between branches/commits and experiments are still starred? I can't see anything that would ever flush stale experiments out of this list. An ever-expanding object is something that we should try to avoid.

Reply:
I don't see a better option tbh, any ideas? I hope this list won't be growing too fast?

It should be probably split by commits for faster access though.